cherrypy/lib/cpstats.py

```diff
@@ -308,6 +308,14 @@ def _get_threading_ident():
     return threading._get_ident()
 
 
+def _get_response_status(resp):
+    try:
+        response_status = resp.output_status
+    except AttributeError:
+        response_status = resp.status
+    return tonative(response_status, 'utf-8')
```

Comment:
`tonative` is deprecated. Let's not use it here. Let's be more precise about what should be expected from response_status and what's needed by `w['Response Status']` and convert to that form. Could this be `return response_status.decode('utf-8')` instead? What reason do we have to believe the value would have been encoded as utf-8?

Reply:
*unicode is hard*

Reply:
I don't have any reason to think that tonative is required here at all, so I'm going to try removing it.